package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint --fix --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "lint": "eslint --ignore-path .eslintignore --ignore-pattern webpack . && kacl lint",
     "pretest": "rimraf ./.nyc_output",
-    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test.*'",
+    "test": "ts-mocha -p tsconfig.json --recursive 'test/**/*.test*'",
```

Comment:
Fixes issue in recently merged PR that skips all tests with `tests`(plural) in there name.

Reply:
Thanks for picking this out, however, it doesn't belong in this PR, I'll submit another PR to address specifically.

Reply:
https://github.com/auth0/auth0-deploy-cli/pull/445/files

Reply:
Cool, I'll rebase on this when it's merged.